experimental/components/Calendar/Microsoft.BotFramework.CustomAction.CalendarSkill.csproj

```diff
@@ -0,0 +1,46 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <!-- Nuget packaging information -->
+  <PropertyGroup>
+    <PackageId>Microsoft.Bot.Components.CalendarSkill</PackageId>
+    <PackageVersion>1.0.0-alpha.1</PackageVersion>
+    <Description>Plugin component for Calendaring custom action</Description>
+    <Summary>Plugin component for Calendaring custom action.</Summary>
+    <Company>Microsoft</Company>
+    <Authors>Microsoft</Authors>
+    <Product>Microsoft Bot Builder SDK</Product>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.11.1" />
+    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Adaptive" Version="4.11.1" />
+    <PackageReference Include="Microsoft.Bot.Builder.Runtime.Plugins" Version="4.12.0-daily.preview.20210201.208375.958cb4f" />
+    <PackageReference Include="Microsoft.Graph" Version="3.15.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\MsGraph\Microsoft.BotFramework.CustomAction.MsGraph.csproj" />
```

Comment:
Not sure if this should be a direct dependency on the msgraph component or a dependency on the published plugin?

Also my namespace comment applies here, but for MsGraph ;)

Reply:
For now I don't know if we want to break out the dependency quite yet... basically if we do that we are forcing calendaring to do an in-branch publishing (change in MsGraph, checkin, publish, then consume).